packages/nx-aws-cache/src/tasks-runner/aws-cache.ts

```diff
@@ -224,7 +251,7 @@ export class AwsCache implements RemoteCache {
 
       return true;
     } catch (err) {
-      if ((err as Error).name === 'NotFound') {
+      if ((err as Error['name']) === 'NotFound') {
```

Comment:
Better to use dot notation as it was before.

Reply:
With dot notation I have compile error

Compiling TypeScript files for project "nx-aws-cache"...
packages/nx-aws-cache/src/tasks-runner/aws-cache.ts:261:19 - error TS2713: Cannot access 'Error.name' because 'Error' is a type, but not a namespace. Did you mean to retrieve the type of the property 'name' in 'Error' with 'Error["name"]'?

261       if ((err as Error.name) === 'NotFound') {

Reply:
`(err as Error).name === 'NotFound'` work perfectly fine, why have you changed it?

Reply:
Change back as you write. I dont know why previous I get syntax error. Maybe yarn format fix this.